server/chat/backend/agent/orchestrator/dispatcher.py

```
@@ -0,0 +1,245 @@
+        logger.debug(
+            "Dispatcher exit: incident_id=%s user_id=%s",
+            incident_id, user_id,
+        )
+        return
+
+    valid_inputs: list[SubAgentInput] = []
+    for raw in raw_inputs:
+        try:
+            valid_inputs.append(SubAgentInput(**raw) if isinstance(raw, dict) else raw)
+        except Exception:
+            logger.exception("dispatcher: invalid SubAgentInput %r — skipping", raw)
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)